services/gas-api/quickstart.md

```diff
@@ -3,20 +3,47 @@ description: Get started with the Gas APIs.
 sidebar_position: 1
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Quickstart
 
 This quickstart shows you how to call the Gas API using JavaScript.
 You can also use a tool such as [cURL](https://curl.se/) or [Postman](https://www.postman.com/) to
 call the REST APIs.
 
 :::tip
-View the [API reference content](api-reference/index.md) to view the `curl` command for each API. 
+You can view the [API reference content](api-reference/index.md) to view the `curl` command for each API. 
 :::
 
+You can call the Gas APIs in two ways:
```

Comment:
Can you move this to the index page of the API reference as in the Infura PR? https://github.com/INFURA/docs/pull/234/files#diff-d9c25f5deab99f3a7c9c90e7ee5e5fdf36f4453d5c4ebeb5a53660dfb4aa38e3R16